main.py

```diff
@@ -134,27 +142,47 @@ def display_score():
         screen.blit(pickup_surf, pickup_rect)
 
         # Player
-        player_rect.x += player_speed
+
+        if accelerating_left:
```

Comment:
Here checked to see whether the player was accelerating, and if it was, add/subtract appropriate velocity to simulate acceleration. Then checked to see if the player wasn't accelerating, and multiplied by a fractional value if true.